core/modules/layout/css/grid-float.css

```diff
@@ -0,0 +1,958 @@
+/*!
+ * Bootstrap v4.0.0-alpha.2 (http://getbootstrap.com)
+ * Copyright 2011-2016 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+ */
+.container,
+.site-width-wrapper {
```

Comment:
yea I don't recognize that class.  Wes's idea I made a prototype of was a
theme setting UI to add the container (Bootstrap site width) class to the
element of choice in dropdown menu, but that was not used.

On Sun, May 15, 2016 at 7:15 PM, Nate Haug <notifications@github.com> wrote:

> In core/modules/layout/css/grid-float.css
> <https://github.com/backdrop/backdrop/pull/1379#discussion_r63300930>:
>
> > @@ -0,0 +1,958 @@
> > +/*!
> > + * Bootstrap v4.0.0-alpha.2 (http://getbootstrap.com)
> > + * Copyright 2011-2016 Twitter, Inc.
> > + * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
> > + */
> > +.container,
> > +.site-width-wrapper {
>
> These .site-width-wrapper classes don't seem be documented in Bootstrap
> and aren't used in our own templates. Did we add this but turn out not to
> need it?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/backdrop/backdrop/pull/1379/files/a90be48e16c24cce9cbfa7cc88c109b8cfce59cf#r63300930>
>